### DISCUSSION TITLE
Problem in Setting up Profile Status

### DISCUSSION THREAD

**Discussion by shreya-paul-17** [community/community#126804] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I tried to update GitHub status by selecting the "octocat" emoji in the emoji picker section and setting "Contributing" in the description section. Later on when I clicked on "Set status" button and I thought it got updated. But it is not showing the expected results. When I am hovering onto the small circle circular section, it is not showing the label "Contributing" whereas in the top right corner when I am clicking on the profile picture, it is showing the description. When I encountered this situation, I thought of putting it back to normal by putting the "Focusing" option which is present in the default Suggestions. 

But this issue is still present. Even when I am hovering below the profile picture, the label "Change your avatar" is showing. As far I know, when we hover over to the space where we have to insert the profile picture, at that time the label should be showing. 

Can anyone please help me how to get through this issue. Any kind of help is highly appreciated from my side. You all can also suggest me any alternative way in order to resolve this issue. Here is my GitHub [link](https://github.com/shreya-paul-17)
You all can also point out if I'm committing any kind of mistake while setting up the profile status in my GitHub. Also please answer me if I can change the border color of the circular section from gray to any other color or not.

**Comment by CopperEagle** [comment]
Hey there @shreya-paul-17 

Unfortunately, I cannot recreate the issue. " :octocat:   Contributing" appeared for me when trying it on my profile.  Also, I am seeing the default "Focussing" when hovering over your profile. Make sure that when you check if your status is updated, open a new tab or at least reload it and only then hover over your name there - likely some caching stuff is going on in your browser.

Also, make sure you set the "reset status" to never ... but you probably did that anyways (it's the default setting).

**Reply by shreya-paul-17** [reply]
Thanks for answering @CopperEagle but unfortunately even after refreshing for several times, I am not able to see "Contributing" as a label when I'm hovering onto that . Instead at that time "octacat" is getting shown up as description. Even after removing the status entirely, "Set status" is also not showing up as a label. According to @davevad93 it is only visible to other people, but could please clarify that when you tried it, were you able to see the label of "Contributing" when you replicated the issue on your system?

**Reply by CopperEagle** [reply]
Oh so I probably misunderstood. The label "Contributing" only appears when you hover over the name in issues, discussions, etc. like here. If you are on the profile and there hover over this status "circle", you will just see ":octacat:" indeed. 

Edit: @davevad93 just posted screenshots in a reply below showing what I meant.  :+1:

**Comment by davevad93** [comment]
Hi @shreya-paul-17 ,

It's not an issue that's the default behaviour, as you can see in the following screenshots, your profile status label is displayed correctly when hovering, the fact is that the label status text on the profile is only visible to oher users, anyway if you hover on your profile's username in this same discussion you can see your status:

![1](https://github.com/community/community/assets/94308234/34fdc902-ed7a-4c47-90f1-eb7db6186962)

![2](https://github.com/community/community/assets/94308234/90d010df-b5a1-493e-9472-8f0cc3522d34)

**Reply by shreya-paul-17** [reply]
Thanks @davevad93 for taking your time and replying to me. However, as far as I remember when I last viewed my profile, and hovered over it, it was showing "Focusing" but as soon as I tried to change it yesterday with "octacat" having the description of "contributing", something wrong has happened. Could please clarify it by replicating this issue and checking it whether you are able to view it or not after setting your status on your own?

**Reply by davevad93** [reply, marked as answer]
Hey @shreya-paul-17  you're welcome, I've set the status `Contributing` using the `:octocat:` emoji and here are the screenshots:

1. When hovering I can see the label but not the text `Contributing`:

![1](https://github.com/community/community/assets/94308234/53bf2969-6fc3-4ac0-bf66-aca4e81f76d5)

2. If I hover on my username the label and its text are visible:

![2](https://github.com/community/community/assets/94308234/52dda12c-f329-4d4f-b389-647852ef9b35)

As you can see the label is there but the status text isn't visible.

**Comment by shreya-paul-17** [comment]
Thanks a lot to both of you (@davevad93  and @CopperEagle) for confirming me in this matter.

**Reply by CopperEagle** [reply]
@davevad93 @shreya-paul-17 

Now I am confused... :joy: the label "Contributing" now appears. :upside_down_face: 
![Screenshot](https://github.com/community/community/assets/137458753/8000323e-fc96-4148-87ca-f1479a8ac1d9)

This could be a backend thing... maybe it requires some time to show up. Github may consider this a low priority data and non-standard labels take some time until updated within the network? Just a guess.

If this is the case, try setting it again and then check back a few hours later - it should appear then.

**Reply by davevad93** [reply]
Hey @CopperEagle , yes the Contributing label appears for other users but not for me, initially i thought that it could be an update thing as you rightly pointed out, but still, I can't see the text when hovering, just the `:octocat:` emoji:

![1](https://github.com/community/community/assets/94308234/d944966f-7380-46fb-ad08-5eace8d35d4b)

P.D. Now I'm going to delete the label, I forgot to do it right after checking its behavior. 😂

**Reply by CopperEagle** [reply]
:thinking: Yeah, I checked again and it really is like that: When creating a custom label, it only appears when not logged into that account. You will be able to see the label when using a seperate browser or incognito mode or logging out...

**Reply by AnweshGangula** [reply]
Yeah, I see the same behavior. I don't see the label when logged in.

**Comment by shreya-paul-17** [comment]
Sorry @CopperEagle for replying you late. I was actually searching in You tube for this issue. See this You tube [video](https://www.youtube.com/watch?v=HD4cnRuSGN0&t=354s) . Watch it from the timestamp 5:30 to 5:45. It is showing exactly the desired outcome which I currently want. However I cannot exactly say what is going on with the custom status that I was trying to set before. But even after setting it with the default "focusing" status among the suggestions, it is not showing this result.

**Reply by CopperEagle** [reply]
Okay so for the standard "focusing" it should show it when hovering above it. What browser & OS are you using? :thinking: Just trying to reduce potential factors here. Though, again, for other accounts / no account, the label will appear when hovering above it. Maybe try logging out or incognito mode or other browser not logged in and check. It could be that when Github engineers implemented this feature, they did not support all possible browsers or something like that.

**Reply by davevad93** [reply]
Hey @shreya-paul-17 , I've watched the video from 5:30 to 5:45 and I noticed that it's from 2022. If you look closely the interface is a bit different, for example in the video the taskbar for `Overview`, `Repositories`, `Projects`, `Packages` and `Stars` is located at the top center while now it is located at the top left. I can't be 100% sure but I guess GitHub has updated the UI so what's shown in the video can't be done anymore.

**Reply by davevad93** [reply]
Hey man @CopperEagle , I've checked this using 3 different browsers in incognito mode, `MS Edge`, `Google Chrome` and `Mozilla Firefox` the behavior is the same. Perhaps as you've pointed out could be an OS related thing. I'm using `Windows` maybe with `Linux` or `MacOS` it's different, I don't know..

**Reply by CopperEagle** [reply]
Hey @davevad93  I can confirm that on `Linux`, for `Mozilla Firefox` it works. Interestingly, for `Chromium` it doesn't. So really, it could be a browser thingy...

Ps. wtf I never knew that "verifying your 2FA configuration" is a thing. After entering the 2FA code on Chromium I had to reenter another 2FA code for that... like who thought of this :joy:

**Reply by davevad93** [reply]
Wut? 😂 that's really odd man, I've logged in with `Firefox` and `Chrome` using an old account of mine without 2FA so I didn't experienced that. I'd never ever imagined that something like that could be possible... "verifying your 2FA configuration" 😂. Anyway thanks for the info, now I know that!

**Reply by shreya-paul-17** [reply]
@CopperEagle I am using Windows and Google Chrome

**Reply by shreya-paul-17** [reply]
@davevad93 Maybe whatever you told is correct, but in the heading of that You tube , I saw "2024 Updated", thus I thought of it in that way. Can you confirm it by putting the default "focusing" status among the suggestions in your status and check whether on hovering the description is getting shown or not?

**Reply by davevad93** [reply]
Hey @shreya-paul-17 , on hovering the text isn't displayed even when using the default status descriptions such as `Focusing`, `On Vacation`, `Out stick`, `Working from home`. I've tested it with `MS Edge`, `Google Chrome` and `Mozilla Firefox` in `Windows`. @CopperEagle said that it works in `Linux` with `Mozilla Firefox` but for `Chromium` based browsers it doesn't. Seems like a mix of `OS` and `Browser` related thing.

**Reply by shreya-paul-17** [reply]
Yes, @davevad93 I also think the same

**Reply by davevad93** [reply]
Update @shreya-paul-17 , looking at the profile of a friend of mine, I've accidentaly hovered the mouse over his status and I was able to see the description, I've taken a look at your profile too and can confirm that now the `Focusing` label description is visible. It must be a backend thing just as @CopperEagle initially said, the GitHub API needs some time to update its data. Something similar could happen with the contribution graph which sometimes needs up to 24 hours to update, what I don't understand is why in `Linux` it's immediatly displayed while in `Windows` you have to wait... but hey, mystery solved.

![1](https://github.com/community/community/assets/94308234/a2111e25-8d2a-4df4-989f-cc97903563a7)

**Reply by shreya-paul-17** [reply]
Yes @davevad93 you can see my description of focusing on my profile but I myself cannot see it for my own GitHub profile

**Reply by davevad93** [reply]
Yes, I'm stupid AF 😂 I forgot the problem was that a user can't see his own status, not sleeping well takes its toll, sorry and have a nice day @shreya-paul-17 !

**Reply by shreya-paul-17** [reply]
It's okay @davevad93 , I can understand...May u also have a great day ahead
